app/src/main/AndroidManifest.xml

```diff
@@ -44,7 +44,8 @@
         <activity
             android:name=".activity.SplashActivity"
             android:screenOrientation="user"
-            android:theme="@style/AppThemeSplash">
+            android:theme="@style/AppThemeSplash"
+            android:exported="true">
```

Comment:
What's the purpose of this one?

Reply:
I added it to resolve an error I encountered after bumping the Android API to 31. You can refer to [b](https://cafonsomota.medium.com/android-12-dont-forget-to-set-android-exported-on-your-activities-services-and-receivers-3bee33f37beb)